src/rgw/rgw_rados.h

```diff
@@ -937,7 +938,7 @@ struct RGWZoneParams : RGWSystemMetaObj {
   int fix_pool_names();
   
   void encode(bufferlist& bl) const {
-    ENCODE_START(8, 1, bl);
+    ENCODE_START(9, 1, bl);
```

Comment:
@cbodley : There was a merge conflict here; tier_config was added in master at version number 8, so i have bumped up the version again to 9, to add roles_pool. Is that correct?